### ISSUE TITLE
Boat Motor Tool Edits

### ISSUE THREAD

**Issue by DawsonDevinity** [GMDevinity/Flood#297]
- [ ] Increase acceleration rate by 15%.
- [ ] Disable simultaneous use of MOUSE2 weapon functionality when operating boat motor
- [ ] When operating tool:  Adjust positioning of boat motor 'ghost' preview so that propeller faces user (i.e., turn 180 degrees from its current orientation).
- [ ] Decrease price from $400 to $200.
- [ ] Introduce an adjustable value range that'd allow users to set the propeller strength of their motor.
- [ ] Allow players to utilize others' spawned boat motors.
- [ ] Enable boat motors to be capable of being used during Build phases.

**Comment by Donkie** [comment]
Sounds good except for disabling mouse2 use on weapons, it's not possible in a easy way. I could change the throttle key to something like E (USE) however.

**Comment by DawsonDevinity** [comment]
Yeah, actually––why not just have the boat motor automatically start from the get-go (that is, after pressing and holding down E) instead of having to press another key to commence start up?
